### PR TITLE
feat: add New Tabs in core [B1M2-3511]

### PR DESCRIPTION
**Pull request by yarema184**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] I have read the [**contributing guidelines**][contributing].
- [ ] My code follows the [code style][code-style] of this project.
- [ ] All new and existing tests passed.
- [ ] My HTML markup is valid and meets [W3C standards](https://validator.w3.org/).
- [ ] My styles avoid hard-coded 'magic' values and make use of the design tokens available in themes.
- [ ] My code meets the [A11Y Web Accessibility Checklist](https://a11yproject.com/checklist). If not, please add justification documentation.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.

[contributing]: https://github.com/coingaming/moon-design/blob/master/CONTRIBUTING.md
[code-style]: https://github.com/coingaming/moon-design/blob/master/CONTRIBUTING.md#code-style

### CODE REVIEW

File: next-docs/pages/components/tabs.tsx [hunk 1]
```diff
@@ -39,27 +88,201 @@ const PreviewTabs = () => {
   );
 };
 
-export default function PageTabs() {
+const PreviewTabsFill = () => {
+  return (
+    <Preview
+      title="Tabs (Filled)"
```
Comment (karl-kallavus) [inline next-docs/pages/components/tabs.tsx:94]: Just "Filled"

Comment (karl-kallavus) [reply next-docs/pages/components/tabs.tsx:94]: here and below

**Review by karl-kallavus — CHANGES_REQUESTED** [review]
minor doc fixes

File: next-docs/pages/components/tabs.tsx [hunk 2]
```diff
@@ -39,27 +88,201 @@ const PreviewTabs = () => {
...
+          TabLink or TabLinkFill components provides the tab interaction.
+        </p>
+        <p className="text-lg mt-4">
+          There are three types of Tabs items:
+          <ol className="ml-4">
+            <li>1. Underline: top or bottom (TabLink);</li>
+            <li>2. Filled (TabLinkFill);</li>
+            <li>3. Segment control (TabLinkFill);</li>
+          </ol>
         </p>
```
Comment (karl-kallavus) [inline next-docs/pages/components/tabs.tsx:186]: Don't think that we need that block at all

**Review by karl-kallavus — CHANGES_REQUESTED** [review]

File: next-docs/pages/components/tabs.tsx (hunk 1 above)
Comment (yarema184) [reply next-docs/pages/components/tabs.tsx]: updated

File: next-docs/pages/components/tabs.tsx (hunk 2 above)
Comment (yarema184) [reply next-docs/pages/components/tabs.tsx:184]: Delete

**Review by karl-kallavus — APPROVED** [review]

**Pull request merged**